README.md

```diff
@@ -1,4 +1,20 @@
 Put the time in weeks.  We'll change the format as need be.  I'm too busy
 putting my kid to sleep to do this nicely ;)
 
+** Current Format Requirements **
+Column and Type:
+
+* Company [string] - Company Name Or Governing Body
+* Paternity [int] - Number of Weeks Paid Paternity Leave
+* Maternity [int] - Number of Weeks Paid Maternity Leave
+* Adoption [int] - Number of Weeks Paid Leave Following an Adoption
+* Notes [string] - Any additional information of note
+* Source [string] - Document Source if Available 
+* Paid/Unpaid [string] - Either 'Paid' or 'Unpaid' as appropriate
```

Comment:
So this is a bit odd - I think I want to scope this to only paid leave.  At least for now.